pkg/controller/controller.go

```diff
@@ -165,24 +168,26 @@ func (c *PodController) deleteObjects(podObj *v1.Pod, parentJobName string, dryR
 	return
 }
 
-
 func (c *PodController) getParentJobName(podObj *v1.Pod, version version.Info) (parentJobName string) {
 
 	oldVersion := false
 
-	major,_ := strconv.Atoi(version.Major)
-	minor,_ := strconv.Atoi(version.Minor)
-	if major < 2 && minor < 8{
+	major, _ := strconv.Atoi(version.Major)
+	re := regexp.MustCompile("[0-9]+")
+	minor, _ := strconv.Atoi(re.FindAllString(version.Minor, -1)[0])
```

Comment:
Could you please add additional check for this regex. It's unlikely to happen, but the code will panic in case `version.Minor` has no numbers at all. 

Something similar to this

```
var minor int
m := re.FindAllString(version.Minor, 1)
if len(minor) != 0 {
   minor, _ = strconv.Atoi(m[0])
} else {
   log.Printf("failed to parse minor version %s", version.Minor)
   minor = 0
}
```

Reply:
Fixed and Tested with empty string. Works

Reply:
thank you